test/TestUserAcceptance.hs

```diff
@@ -13,13 +14,19 @@ import System.IO (hPutStr, stderr)
 import System.Environment (lookupEnv)
 import Control.Monad (void, when)
 import Data.Maybe (isJust)
+import Control.Monad.IO.Class (liftIO)
+import Control.Monad.Reader (runReaderT, ask, ReaderT)
 
+import Control.Lens.Getter (view)
+import Control.Lens.Tuple (_3)
```

Comment:
Can we just import `view` and `_3` from the top-level module?

```haskell
import Control.Lens (_3, view)
```

Reply:
Ah... did not know ... cheers!